setup.py

```diff
@@ -52,7 +52,7 @@
 DOWNLOAD_URL = "https://github.com/pydicom/pydicom/archive/master.zip"
 LICENSE = "MIT"
 VERSION = __version__
-# PACKAGE_DATA = {'pydicom': ['../tests/tests_files/*']}
+PACKAGE_DATA = {'pydicom': ['tests/test_files/*', 'tests/charset_files/*']}
 REQUIRES = []
```

Comment:
To me its main function is what you call **validation** which I need to do somewhere, and since I find harder to work with windows and exporting the data on the setup facilitates my windows testing. I went for it (I kill 2 birds, single commit) 
Once said that, my original idea is to move the tests to use the data of the Examples (or generate the data within the test when generating invalid data for tests.  ie: the inline snipped within the **issue template** )